retroshare-gui/src/gui/common/NewFriendList.cpp

```diff
@@ -763,8 +761,8 @@ void NewFriendList::peerTreeWidgetCustomPopupMenu()
     if (RSLinkClipboard::empty(RetroShareLink::TYPE_CERTIFICATE))
         action->setDisabled(true);
 
-    contextMenu.addAction(FilesDefs::getIconFromQtResourcePath(IMAGE_EXPAND), tr("Expand all"), ui->peerTreeWidget, SLOT(expandAll()));
-    contextMenu.addAction(FilesDefs::getIconFromQtResourcePath(IMAGE_COLLAPSE), tr("Collapse all"), ui->peerTreeWidget, SLOT(collapseAll()));
+    contextMenu.addAction(QIcon(""), tr("Expand all"), ui->peerTreeWidget, SLOT(expandAll()));
```

Comment:
Why remove the menu icon?

Reply:
i can add new ones, i thinked this icons no needed

Reply:
they are needed since the code was using them! Maybe they do not show in windows? On linux they show in front of each drop menu entry